app/services/deploy_service.rb

```diff
@@ -61,6 +71,20 @@ def github_access_token
       (raise 'A profile and basic_password are required for Github authentication')
   end
 
+  def deliver_slack_webhook(pull_request, webhook_url)
+    uri = URI.parse webhook_url
+    request = Net::HTTP::Post.new(uri.request_uri)
+    request['Content-Type'] = 'application/json'
+    request.body = {
+      text: "The following changes will be released shortly: #{pull_request.html_url}"
```

Comment:
If we are hardcoding the time, we should probably state `in one hour` rather than `shortly`.